pkg/orchestrator/config.go

```diff
@@ -178,6 +181,14 @@ func LoadConfig() (*Config, error) {
 	default:
 		providerKind = models.AWS
 	}
+	provider := models.Provider{
+		DisplayName: utils.PointerTo(string(providerKind)),
+		Status: &models.ProviderStatus{
+			State:              models.ProviderStatusStateUnknown,
+			Reason:             models.NoHeartbeatReceived,
+			LastTransitionTime: time.Now(),
+		},
+	}
```

Comment:
I don't think we should be creating the provider in the configuration, we should be taking in the provider display name as a config input, I think defaulting to "vmclarity-orchestrator", we can pass through the CFN stack name or something in the installers.
The provider object itself should be created as part of the orchestrator boot up flow using the config.